presto-docs/src/main/sphinx/release/release-0.212.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.212
+=============
+
+General Changes
+---------------
+
+* Add the :func:`ST_LineString` function to form a ``LineString`` from an array of points.
```

Comment:
Try to follow this order: fix, improve/optimize, add

Remove "the" to be consistent with other items mentioning functions